<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>594</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="4" column="0">
      <widget class="QSplitter" name="splitter_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QCheckBox" name="Online_Plt_checkBox">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Meas_pushButton">
        <property name="text">
         <string>Measure</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSplitter" name="Input_Splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="Input_Lab_1">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_1"/>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="Input_Lab_2">
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Step</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="lineEdit_3">
         <property name="text">
          <string>End</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="Input_Lab_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSplitter" name="Mode_Sel_Splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QRadioButton" name="Step_radioButton">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="Num_radioButton">
        <property name="text">
         <string>Num</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QProgressBar" name="MeasProgressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Avg Times</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_AvfTime">
         <property name="text">
          <string>16</string>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>dt</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="text">
          <string>0.01</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>761</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_plot"/>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>9</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_data">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item row="0" column="1">
      <widget class="QSplitter" name="splitter_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="data_save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="data_clear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
